metadata.yaml

```
@@ -0,0 +1,25 @@
+# Copyright 2022 Canonical
+# See LICENSE file for licensing details.
+
+name: tls-certificates-interface
+description: |
+  Placeholder charm
+summary: |
+  Placeholder charm
+
+containers:
+  placeholder:
+    resource: placeholder-image
+
+resources:
+  placeholder-image:
+    type: oci-image
+    description: Placeholder for OCI image
+
+provides:
+  certificates:
+    interface: tls_certificates
+
+requires:
+  certificates:
+    interface: tls_certificates
```

I think that if you change this to
```suggestion
provides:
  certificates-provider:
    interface: tls_certificates

requires:
  certificates-requirer:
    interface: tls_certificates
```
then you'd be able to deploy two instances of the charm and relate them to each other, for testing purposes.